### ISSUE TITLE
dogecoin moving up a lot, whole-coin wont make sense much longer

### ISSUE THREAD

**Issue by buzztiaan** [mniip/Doger#9]
we're over 0.1 usdcent per doge at the moment, prices seem to not follow the 'just round it down' model that was suggested many years ago.

future will probably see 1 usd be under 100 doge much quicker than the founders anticipated, the bot should follow suit and not block usage like now.

**Comment by randolf** [comment]
It looks like fractional amounts are supported in the irc.freenode.net#dogecoin channel now (and for at least a week or two), so this issue seems to be resolved.

**Comment by buzztiaan** [comment]
by replacing Doger with something else ;)

**Comment by randolf** [comment]
@buzztiaan Oh.  What has Doger been replaced with?

**Comment by buzztiaan** [comment]
by @Bill48105 's multicoin bot, afaik non-foss

**Comment by Bill48105** [comment]
as @buzztiaan mentioned my "Wallet" (formerly DogeWallet) bot not only supports partial coins but also MULTIPLE coins within a single bot. It is a fork of Doger which I've shared a very small subset of my modifications to github (adding basic soak & few supporting commands like active & whogot) that are backward compatible as I didn't want to break that for Doger. (For example on the fork, although not ideal, I opted to save the activity list to file so as not need to modify the database tables like I did on my bot).  The goal was to remain 100% backward compatible in hopes they'd be merged with Doger & become part of the official #dogecoin tipbot but he declined.  If there is sufficient interest I might spend the time to merge my partial coin changes I've done in Wallet to my fork but have no idea if anyone even uses Doger or my fork to bother.
